### ISSUE TITLE
About center op

### ISSUE THREAD

**Issue by ChrisMii** [EncodeTS/TensorFlow_Center_Loss#9]
whats the difference between training  trainable  centers jointly with major losses and update untrainable centers independently?  ? I think, in  joint way, the trainable centers will be updated as the independent way except the latter can have a different learning rate.
